<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/pykitchen/pykitchen.png);</string>
  </property>
  <widget class="QPushButton" name="cocinarButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cocinar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="despacharButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Despachar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cargarButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guardarButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pykitchen.qrc"/>
 </resources>
 <connections/>
</ui>
